error/s2n_errno.h

```diff
@@ -297,6 +297,7 @@ extern __thread const char *s2n_debug_str;
 #endif /* CBMC */
 
 #define S2N_IMPLIES(a, b) (!(a) || (b))
+#define S2N_IFF(a, b) (S2N_IMPLIES(a, b) && S2N_IMPLIES(b, a))
```

Comment:
Can you add some documentation around this? I actually didn't know what `IFF` meant 😄 and had to [look it up](https://en.wikipedia.org/wiki/If_and_only_if).

Reply:
Of course! I added a comment/note.